test/shared/src/main/scala/zio/test/ZIOSpec.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.test
+
+import org.portablescala.reflect.annotation.EnableReflectiveInstantiation
+import zio._
+import zio.test.environment.TestEnvironment
+import zio.test.render._
+
+@EnableReflectiveInstantiation
```

Comment:
This will only work with a non-polymorphic trait. Hence the need to have a dummy parent without type parameters (only type members).

Reply:
Done.